fuel-vm/src/checked_transaction/builder.rs

```diff
@@ -20,27 +20,35 @@ where
     Tx: IntoChecked,
 {
     /// Finalize the builder into a [`Checked<Tx>`] of the correct type
-    fn finalize_checked(&mut self, height: BlockHeight) -> Checked<Tx>;
+    fn finalize_checked(&mut self, height: BlockHeight, gas_price: u64) -> Checked<Tx>;
```

Comment:
It seems `gas_price` can be part of the builder as well along with `block_height`

Reply:
`gas_price` _isn't_ part of the transaction, though. It's part of the block.

Reply:
It's true, but this type is used only in tests, and for simplicity it could be better=) But it is up to you since you need to update all palces in the `fuel-core`=)

Reply:
I think I prefer it this way. I'll just have to deal with any costs on the `fuel-core` side :)